system/lib/libcxxabi/src/cxa_guard.cpp

```diff
@@ -9,6 +9,29 @@
 
 #include "abort_message.h"
 
+#if __EMSCRIPTEN__
+
+// XXX EMSCRIPTEN making guard operations simple and LTO-optimizable opens up
```

Comment:
Oh, thanks, now I understand. Does libcxx only ever call `__cxa_guard_acquire`, `__cxa_guard_release` `__cxa_guard_abort` from from application startup time global ctors and nowhere else? Tried searching info about these but couldn't find anything. Are these functions documented somewhere?